h/static/scripts/directive/test/sidebar-tutorial-test.js

```diff
@@ -9,7 +9,7 @@ describe('SidebarTutorialController', function () {
       var session = {
         state: {
           preferences: {
-            show_sidebar_tutorial: true
+            'show_sidebar_tutorial': true
```

Comment:
Fair enough, although I will say that just because there are no violations in the current codebase doesn't mean that it hasn't come up in code reviews. In this case though, I couldn't find any PR comments that have mentioned it either so I have removed this commit.